akka-docs/src/main/paradox/remoting-artery.md

```diff
@@ -135,6 +135,12 @@ Aeron transport and less risk of trouble in container environments.
 
 @@@ note
 
+@ref:[Rolling update](additional/rolling-updates.md) is not supported when changing from one transport to another.
+
+@@@
```

Comment:
A bit weird with two note-blocks after each other, does it need to be a note callout or could it be just a regular text section?

Reply:
is that good to go or we still plan to change this?

Reply:
I removed the note around the "Aeron 64 bit"